### PR TITLE
Add download link to item row

### PR DESCRIPTION
**Pull request by mgrauer**
An attempt to bring @AlmightyYakob 's changes from #447 to current master.

### CODE REVIEW

File: web/src/views/FileBrowserView/FileBrowser.vue [hunk 1]
```diff
@@ -55,6 +68,8 @@ import {
   getLocationFromRoute,
 } from '@/utils';
 
+import { girderRest } from '@/rest';
```
Comment (AlmightyYakob) [inline web/src/views/FileBrowserView/FileBrowser.vue:71]: In this branch `rest.js` exports this by default:
```suggestion
import girderRest from '@/rest';
```

**Comment by mgrauer** [comment]
Only one error left

https://deploy-preview-448--gui-dandiarchive-org.netlify.app/#/file-browser/folder/5e6eb2b776569eb93f451f8d

**Comment by mgrauer** [comment]
Actually the item listing page looks ok, but there is an error on the dandiset landing page, though that is also in production so unrelated to this branch.

**Comment by mgrauer** [comment]
I'd like to merge this into master as a quick fix for https://github.com/dandi/dandiarchive/issues/449, though there is a PR for this going into publish which will also solve it.

**Comment by satra** [comment]
thanks for getting this in.

**Pull request merged**